data/Gibraltar/Parliament/sources/instructions.json

```diff
@@ -0,0 +1,59 @@
+{
+  "sources": [
+    {
+      "file": "morph/data.csv",
+      "create": {
+        "type": "morph",
+        "scraper": "andylolz/gibraltar-parliament",
+        "query": "SELECT * FROM data"
+      },
+      "source": "http://www.parliament.gi/history/composition-of-parliament",
+      "type": "membership"
+    },
+    {
+      "file": "morph/terms.csv",
+      "create": {
+        "file": "morph/terms.csv",
```

Comment:
there's no need to duplicate this line to here — the `create` destination will always be the `file` (I expect you just copied this from somewhere else that also makes the same mistake, but you may as well remove it)